### ISSUE TITLE
[Bug]: allowCommercialUse is now returning []enum and this is not documented

### ISSUE THREAD

**Issue by drgrib** [civitai/civitai#1376]
### What happened?

Get requests to `https://civitai.com/api/v1/models` are now returning responses with `allowCommercialUse` as an `[]enum` rather than a single `enum` value as documented.

```json
...
			"allowCommercialUse": [
				"Image",
				"RentCivit",
				"Rent"
			],
...
```

### Steps to reproduce the problem

1. Get request to `https://civitai.com/api/v1/models`
2. Look at the result for `allowCommercialUse`

### What should have happened?

Document this. I would have done it myself but there is no way for me to edit the wiki.

### What platforms do you use to access the site?

MacOS

### What browsers do you use to access the site?

Google Chrome

**Comment by sovos15** [comment]
was this [Fixed](https://github.com/jkawamoto/go-civitai/issues/6)?

**Comment by drgrib** [comment]
No. This is still not documented. `allowCommercialUse` is documented as returning a single `enum`, not an `enum[]`:
https://github.com/civitai/civitai/wiki/REST-API-Reference#get-apiv1models

However, this is just one of many undocumented changes that you can see if you use the API interface you linked. It seems to me that proper documentation of the API has basically been abandoned and projects like the one you linked are the only way to discover what is actually being returned by API calls.

**Comment by sovos15** [comment]
At the very least they could use the tool themselves to keep their own documentation up to date. Quite miffed about the state of metadata too. Beyond lazy, many projects attempted to format metadata for Civitai and they somehow keep updating to not be able to read that. I really do worry it's being destroyed from the inside.

**Comment by manuelurenah** [comment]
Sorry for the delay guys, this has been fixed and should be out already! Cheers!
